File: DemoApp-Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/data/main/SectionHeader.java
package com.applovin.enterprise.apps.demoapp.data.main;

/**
 * A {@link ListItem} representing a section header on the main screen
 * <p>
 * Created by santoshbagadi on 2019-09-10.
 */
public class SectionHeader
        implements ListItem
{
    private final String title;

    public SectionHeader(final String title)
    {
        this.title = title;
    }

    /**
     * @return The time of the section header.
     */
    public String getTitle()
    {
        return title;
    }

    @Override
    public int getType()
    {
        return TYPE_SECTION_HEADER;
    }
}

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/data/main/SectionHeader.kt
package com.applovin.enterprise.apps.demoapp.data.main

/**
 * A [ListItem] representing a section header on the main screen
 * <p>
 * Created by Harry Arakkal on 9/17/2019.
 */
data class SectionHeader(val title: String, override val type: Int = ListItem.SECTION_HEADER) : ListItem
